docs/developing/app-startup.md

```diff
@@ -0,0 +1,76 @@
+# Ways in which xPub Starts
+
+## Dockerfile(s)
+
+In general the actual Dockerfiles don't specify the way in which the application
+starts (see below).
+
+```
+~/dev/elife-xpub $ grep CMD Dockerfile*
+Dockerfile:CMD []
+Dockerfile-development:CMD []
+```
+
+However, because the `Dockerfile` is meant for developers it does run the application
+explcitly with `RUN [ "npx", "pubsweet", "build"]`
+
+## Docker Compose
+
+In general docker-compose is used as the way to start the application.
```

Comment:
`In general` repeated as the first two words on the first two sections. This one could be replaced with `docker-compose can be`

Reply:
@will-byrne changed... please :eyes: